static/src/js/components/CustomTextWidget/CustomTextWidget.jsx

```diff
@@ -103,7 +108,7 @@ const CustomTextWidget = ({
         placeholder={placeholder}
         ref={focusRef}
         tabIndex={0}
-        type={type && type === 'number' ? 'number' : 'text'}
+        type="text"
```

Comment:
So I'm guessing there were no use cases that you found in which we used CustomTextWidget as a number?

Reply:
CustomTextWidget is still being used for number field, but not as html element input type="number" because Firefox and Safari have issues with it. See method handleChange:
if (type === 'number') {
      valueToSet = newValue.replace(/[^\d.eE+-]/g, '')
    }